src/hooks/useUserProfile.test.tsx

```diff
@@ -1,4 +1,3 @@
-// import React from 'react';
```

Comment:
Should we change the file extension?

Reply:
Looks like there are a few `.tsx` that could be `.ts` - I'll grab them all in a separate PR